yt_dlp/extractor/cda.py

```diff
@@ -378,6 +378,9 @@ class CDAFolderIE(InfoExtractor):
                 'title': 'TESTY KOSMETYKÓW',
             },
             'playlist_mincount': 139,
+        }, {
+            'url': 'https://www.cda.pl/Test-t3st_t-1/folder/12345678',
```

Comment:
Can we use a real URL please? I know it's `only_matching`, but a real world example is still much more useful

Reply:
```suggestion
            'url': 'https://www.cda.pl/FILMY-SERIALE-ANIME-KRESKOWKI-BAJKI/folder/18493422',
```